sdk/python/feast/client.py

```diff
@@ -394,12 +404,15 @@ def list_projects(self) -> List[str]:
 
         """
 
-        response = self._core_service.ListProjects(
-            ListProjectsRequest(),
-            timeout=self._config.getint(opt.GRPC_CONNECTION_TIMEOUT),
-            metadata=self._get_grpc_metadata(),
-        )  # type: ListProjectsResponse
-        return list(response.projects)
+        if self._use_object_store_registry:
+            raise NotImplementedError("Projects are not implemented for object store registry.")
```

Comment:
How will we accommodate teams that already have projects implemented. Will the migration script remove the project prefix?